lib/thredded/collection_to_strings_with_cache_renderer.rb

```diff
@@ -95,10 +95,11 @@ def render_partials(view_context, collection:, render_threads:, **opts)
     # @param view_context
     # @return [Array<String>]
     def render_partials_serial(view_context, collection, opts)
-      partial_renderer = ActionView::PartialRenderer.new(@lookup_context)
-      collection.map { |object| render_partial(partial_renderer, view_context, opts.merge(object: object)) }
+      collection.map do |object|
+        partial_renderer = ActionView::PartialRenderer.new(@lookup_context, locals: { post: object })
+        partial_renderer.render(opts[:partial], view_context, nil).body
```

Comment:
I think the `nil` should be `opts` (I think we're never calling this with any optional keyword arguments in `render_collection_to_strings_with_cache`, so it won't raise an error, but if we have the **opts there, then we need to respect them)